src/helm/benchmark/presentation/summarize.py

```diff
@@ -161,22 +168,32 @@ def get_method_display_name(model_display_name: Optional[str], info: Dict[str, A
     return (model_display_name or "???") + (f" [{dict_to_str(info)}]" if len(info) > 0 else "")
 
 
-def compute_average_row_win_rates(table: Table) -> List[Optional[float]]:
+def compute_aggregate_row_win_rates(table: Table, aggregation: str = "mean") -> List[Optional[float]]:
     """
-    Computes the average win rate of each row across columns. For a given row r1, if we pick another row r2 and column
-    c1 uniformly at random, what is the probability that r1c1 is better that r2c1? We skip columns where "better" is
-    ambiguous or less than 2 values are non-null.
-    Returns a list of average win rates, one per row, with None if a row was never meaningfully comparable (i.e., all
+    Computes the aggregate win rate of each row across columns. For a given row r1 and column c1, the win rate of r1 wrt
+    to c1 corresponds to: if we pick another row r2 uniformly at random, what is the probability that r1c1 is better
+    that r2c1?
+    `aggregation` determines how we aggregate win rates across columns, currently can be "mean" or "median".
+    We skip columns where "better" is ambiguous or less than 2 values are non-null.
+    Returns a list of aggregate win rates, one per row, with None if a row was never meaningfully comparable (i.e., all
     non-null values of the row are in columns we skip).
     """
+    assert aggregation in ["mean", "median"]
     win_rates_per_row: List[List[float]] = [[] for _ in table.rows]
     for i, header_cell in enumerate(table.header):
         lower_is_better = header_cell.lower_is_better
         if lower_is_better is None:  # column does not have a meaningful ordering
             continue
 
         # sort row indices by cell value and then compute the number of wins as the index in the sorted list
-        values = [(row[i].value, j) for j, row in enumerate(table.rows) if row[i].value is not None]
+        def is_cell_valid(cell: Cell) -> bool:  # ignore cells which are strongly contaminated or have no value
+            if cell.value is None:
+                return False
+            if cell.description and CONTAMINATION_SYMBOLS[CONTAMINATION_LEVEL_STRONG] in cell.description:
```

Comment:
This is a hacky because we are assuming that the description has the string corresponding to strong contamination.  Could we add contamination level as a field in `Cell`?

Reply:
Makes sense, added contamination at Cell level.